sys/usb/Kconfig

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2019 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_USB
+    bool "Configure USB"
+    depends on MODULE_USBUS
+    help
+        Configure the USB peripheral via Kconfig.
+
+if KCONFIG_USB
+
+menu "Power management"
```

Comment:
```suggestion
 menu "Configure USB Power management" 
```

Reply:
I'm not really sure about this one. I'm adding `Configure <module>` in `menuconfig` entries because it's an special case, where the user actually enables the configuration of the module. But for other cases it may be a bit redundant to add `configure <feature>` or `enable <feature>` (for booleans).

Reply:
Okay!